packages/cli/src/resolve-options.ts

```diff
@@ -0,0 +1,256 @@
+import { nodeFs } from '@file-services/node';
+import type { Arguments } from 'yargs';
+import yargs from 'yargs';
+import { createGenerator } from './build';
+import type { ConfigOptions, PartialConfigOptions } from './projects-config';
+
+const { join } = nodeFs;
+
+export interface CliArguments {
+    rootDir: string;
+    srcDir: string | undefined;
+    outDir: string | undefined;
+    esm: boolean | undefined;
+    cjs: boolean | undefined;
+    css: boolean | undefined;
+    stcss: boolean | undefined;
+    dts: boolean | undefined;
+    dtsSourceMap: boolean | undefined;
+    useNamespaceReference: boolean | undefined;
+    namespaceResolver: string;
+    injectCSSRequest: boolean | undefined;
+    cssFilename: string | undefined;
+    cssInJs: boolean | undefined;
+    optimize: boolean | undefined;
+    minify: boolean | undefined;
+    indexFile: string | undefined;
+    manifest: boolean | undefined;
+    manifestFilepath: string;
+    customGenerator: string | undefined;
+    ext: string | undefined;
+    require: string[];
+    log: boolean | undefined;
+    diagnostics: boolean | undefined;
+    diagnosticsMode: string | undefined;
+    watch: boolean;
+}
+
+export function getCliArguments(): Arguments<CliArguments> {
+    const defaults = createDefaultOptions();
+    return yargs
+        .usage('$0 [options]')
+        .option('rootDir', {
+            type: 'string',
+            description: 'root directory of project',
+            default: process.cwd(),
```

Comment:
move to defaults?

Reply:
You can not configure the `rootDir` from the config. That's why it can not be in the defaults of the configuration options.